FWCore/Services/test/test_sitelocalconfig.sh

```diff
@@ -6,6 +6,9 @@ function die { echo $1: status $2 ;  exit $2; }
 mkdir -p ${CMSSW_BASE}/test/SITECONF
 mkdir -p ${CMSSW_BASE}/test/SITECONF/local
 mkdir -p ${CMSSW_BASE}/test/SITECONF/local/JobConfig
+
+export SITECONFIG_PATH=${CMSSW_BASE}/test/SITECONF/local
```

Comment:
I would rather replace the `export CMS_PATH` on line 12/15 with this line. With this PR the `SiteLocalConfigService` no longer uses `CMS_PATH`.

Reply:
Done